dhis-2/dhis-api/src/main/java/org/hisp/dhis/message/MessageConversation.java

```diff
@@ -148,13 +148,15 @@ public void addMessage( Message message )
         if ( message != null )
         {
             message.setAutoFields();
-        }
 
-        this.messages.add( message );
```

Comment:
This changes the behavior int that we now will not add `null` messages. As I don't know what this code is for it might make sense, but might also not.